### PR TITLE
feat: add `rush` spec

### PR DESCRIPTION
**Pull request by F3n67u**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

feature

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

Add spec for rushjs https://rushjs.io/

**Additional info:**

### CODE REVIEW

File: src/rush.ts [hunk 1]
```diff
@@ -0,0 +1,840 @@
...
+            "The position in the word to be completed. The default value is 0",
+        },
+      ],
+    },
+  ],
+  options: [
+    {
+      name: ["-h", "--help"],
+      description: "Show this help message and exit",
+    },
```
Comment (fedeci) [inline src/rush.ts:832]: Adding `isPersistent` will add this option to subcommands too. We can remove those.
```suggestion
    {
      name: ["-h", "--help"],
      description: "Show this help message and exit",
      isPersistent: true,
    },
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Just one small change and we are ready to merge! The spec looks awesome!

File: src/rush.ts (hunk 1 above)
Comment (F3n67u) [reply src/rush.ts]: Thanks for the review. I also removed the redundant help option declaration for subcommands.

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**